src/librustc_typeck/check/method/suggest.rs

```diff
@@ -82,34 +82,63 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         let print_disambiguation_help = |
```

Comment:
Time to move this closure out to a proper method due to size?

Reply:
done